DuckDuckGo/Statistics/GeneralPixel.swift

```diff
@@ -780,9 +780,9 @@ enum GeneralPixel: PixelKitEventV2 {
         case .dbInitializationError:
             return "dbie"
         case .dbSaveExcludedHTTPSDomainsError:
-            return "dbsw"
+            return "database_save_excluded_https_domains_error"
         case .dbSaveBloomFilterError:
-            return "dbsb"
+            return "database_save_bloom_filter_error"
```

Comment:
As part of the Privacy Triage, I'm updating this with clear naming. Once the PR is merged I'll update the macOS dashboard to sum the old and new pixel name, and announce the change in case anyone else on the team knows of a place that should be updated.